usecases/scheduledexecution/export_schedule_execution.go

```diff
@@ -122,6 +144,13 @@ func (exporter *ExportScheduleExecution) ExportDecisions(ctx context.Context, sc
 	var number_of_exported_decisions int
 
 	for decision := range decisionChan {
+		if decision.OrganizationId != organizationId {
```

Comment:
To be more precise and expand on the comment above (repositories/decisions_repository.go), this makes sure in the usecases layer we don't send a decision to an org it doesn't belong to - but I'm not totally sure what happens with the channel if we return here. Probably if we return an error the context will end up being canceled higher up and it's fine, but I'd like to double check. I made a note on linear.
(We're not supposed to close the channel from the receiver side)